src/Gui/CommandView.cpp

```diff
@@ -3330,6 +3332,38 @@ bool StdCmdMeasureDistance::isActive()
     return false;
 }
 
+//===========================================================================
+// Std_Measure
+// this is the Unified Measurement Facility Measure command
+//===========================================================================
+
+
+DEF_STD_CMD_A(StdCmdMeasure)
+
+StdCmdMeasure::StdCmdMeasure()
+  :Command("Std_Measure")
+{
+    sGroup        = "Measure";
+    sMenuText     = QT_TR_NOOP("&Measure");
+    sToolTipText  = QT_TR_NOOP("Measure a feature (UMF)");

```

Comment:
I think the (UMF) from the tooltip should be removed as for a user it's not clear what it means and for future translators it will cause some confusion, too.